db/history.go

```diff
@@ -0,0 +1,211 @@
+package db
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	whisper "github.com/status-im/whisper/whisperv6"
+	"github.com/syndtr/goleveldb/leveldb/util"
+)
+
+var (
+	// ErrEmptyKey returned if key is not expected to be empty.
+	ErrEmptyKey = errors.New("TopicHistoryKey is empty")
+)
+
+// Interface is a common interface for DB operations.
+type Interface interface {
```

Comment:
I would say that when called from outside `db.DB` looks better than `db.Interface`

Reply:
done